frameworks/PHP/peachpie/Website/db.php

```diff
@@ -13,10 +13,10 @@ function db() {
   mysql_select_db('hello_world', $link);
 
   $id = mt_rand(1, 10000);
-  $result = mysql_query("SELECT randomNumber FROM World WHERE id=$id", $link);
+  $result = mysql_query("SELECT id,randomNumber FROM World WHERE id=$id", $link);
 
   // Store result in array.
-  echo json_encode( ['id' => $id, 'randomNumber' => mysql_result($result, 0, 0)] );
+  echo json_encode( ['id' => $id, 'randomNumber' => mysql_result($result, 0, 1)] );
```

Comment:
The id must be read from the db too.

Reply:
I see, will be fixed. fyi php/hhvm (https://github.com/TechEmpower/FrameworkBenchmarks/blob/d454e5062b4742ae6c3eab230d042adba5c52a9f/frameworks/PHP/hhvm/once.php.inc#L39) is wrong as well, although it is tagged as "broken"

Reply:
Need to be changed too.
Thank you

Reply:
When anybody try to repair the hhvm, will be enforced to change it too.